lib/Logger/Logger.h

```diff
@@ -7,22 +7,20 @@
 class Logger {
     public:
         enum class Level : int {
-            TRACE = 0,
-            DEBUG = 1,
-            INFO = 2,
+            TRACE   = 0,
+            DEBUG   = 1,
+            INFO    = 2,
             WARNING = 3,
-            ERROR = 4,
-            FATAL = 5,
-            SILENT = 6,
+            ERROR   = 4,
+            FATAL   = 5,
+            SILENT  = 6,
```

Comment:
looks like some kinde of alignment between lines is going on here.

Reply:
If it's done automatically and somewhat consistently I don't really have an issue with it. Hand-formatting that stuff is incredibly tedious, though.

Reply:
If you have enabled `pre-commit` it is automatic. `pio run -t format` also fixes this automatically.